portfolio/src/main/webapp/index.html

```diff
@@ -87,10 +87,30 @@ <h4>Fractal Project in action</h4>
           <h3 class="padding-above">Find me here</h3>
           <div id="map"></div>
         </div>
-        <div id="contact">
-          <!--TODO: Add comments section -->
-        </div>
       </div>
     </div>
+    <div id="contact">
+      <h3>Contact me here</h3>
+      <form method="POST" action="/comments">
+        <p>Name</p>
+        <input type="text" name="name-input" placeholder="Your Name"/>
+        <br/><br/>
```

Comment:
I'm assuming these blank lines are to increase the space between these elements. It's better to add a class to the p tags and do this in css. Same for below.

Reply:
I just got rid of them because there seemed to be too much extra space anyways and they were not serving a great purpose